Exercise01/Program.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Web;
+using Newtonsoft.Json;
+using System.Threading;
+
+namespace Exercise01
+{ 
+    class Program
+    {
+        public static string ApiKey = "087498271007cb77b740b52dc9a1b4ad-us17";
+        public static string IdList = "2a02210045";
+        public static string IdCompaign = "90e5c4fc9c";
+
+        static void Main(string[] args)
+        {
+            Console.ReadKey();

```

Comment:
Please change:
- The program should allow user enter some content and an email address then send the email.
- Try to use retrofit.net library

Reply:
Done